### DISCUSSION TITLE
15 May 2023 Fine Tuning Tutorial

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [neonbjb/tortoise-tts#443] [Show and tell]
Dear @neonbjb, Could you also add this tutorial to the relevant section of the readme file please? Thank you for this amazing repo 

[**Master Deep Voice Cloning in Minutes: Unleash Your Vocal Superpowers! Free and Locally on Your PC**](https://youtu.be/OiMRlqcgDL0)

[![image](https://github.com/152334H/DL-Art-School/assets/19240467/1a5b1d93-7fc8-473b-b88a-d44beee105a9)
](https://youtu.be/OiMRlqcgDL0)

This tutorial is based on

Ozen Toolkit for data preprocessing
DLAS for Training
Tortoise TTS Fast for speech synthesis

**Comment by noahschulman14** [comment]
Is this significantly better than just using tortoise-tts?
